modules/tables/table_12_Ujjwal_Tiwari_.html

```diff
@@ -0,0 +1,94 @@
+<style>
+    pre{
```

Comment:
Use precise class names so that your module does not affect another module or outer elements using this tag